CHANGES.rst

```diff
@@ -30,14 +31,22 @@ Internals
 
 #. ``boxes_to_`` methods are now optional for ``BoxElement`` subclasses. Most of the code is now moved to the ``mathics.format`` submodule, and implemented in a more scalable way.
 #. ``mathics.builtin.inout`` was splitted in several modules (``inout``, ``messages``, ``layout``, ``makeboxes``) in order to improve the documentation.
-# ``0`` with a given precision (like in ```0`3```) is now parsed as ``0``, an integer number. 
+#. mpmath conversion now supports a new parameter to set the accuracy of the number.
```

Comment:
mpmath conversion means `from_mpath()`, right? If so I'd say that to be clearer. "conversion" could mean from something to mpmath or mpmath to something, and I'dd give the name of the parameter like `acc` so something like: 

> #. ``from_mpmath()`` supports new parameter ``acc`` to set the accuracy of the mpmath number.